src/sentry/sentry_metrics/configuration.py

```diff
@@ -108,6 +110,24 @@ def get_ingest_config(
             )
         )
 
+        _register_ingest_config(
+            MetricsIngestConfiguration(
+                db_backend=IndexerStorage.EXPERIMENRAL,
+                db_backend_options={},
+                input_topic=settings.KAFKA_INGEST_PERFORMANCE_METRICS,
+                output_topic=settings.KAFKA_SNUBA_GENERIC_METRICS,
+                use_case_id=UseCaseKey.EXPERIMENRAL,
+                internal_metrics_tag="rebalance-exp",
+                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS_PERFORMANCE,
+                writes_limiter_namespace=PERFORMANCE_PG_NAMESPACE,
+                cardinality_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_CARDINALITY_LIMITER_OPTIONS_PERFORMANCE,
+                cardinality_limiter_namespace=PERFORMANCE_PG_NAMESPACE,
+                is_output_sliced=settings.SENTRY_METRICS_INDEXER_ENABLE_SLICED_PRODUCER,
+                should_index_tag_values=False,
+                schema_validation_rule_option_name=GENERIC_METRICS_SCHEMA_VALIDATION_RULES_OPTION_NAME,
```

Comment:
@dbanda We might need to create our own option for these settings, feel free to contact ops and update them here

Reply:
Do we want to share the cardinality/writes limiter namespaces (my guess is no?)?

Reply:
Probably not, but since cardinality limiter is disabled and we're not really using pg indexer I might just have them point to the same thing